src/plasma/plasma_manager.h

```diff
@@ -162,6 +163,10 @@ typedef struct PlasmaRequestBuffer {
   int64_t data_size;
   uint8_t *metadata;
   int64_t metadata_size;
+  bool started = false;
+  bool complete = false;
```

Comment:
I wonder if `started` and `complete` should be part of an enum with several different states?

Reply:
:+1: I like this idea